packages/calcite-components/src/components/modal/modal.tsx

```diff
@@ -532,7 +532,7 @@ export class Modal
     this.contentId = ensureId(contentEl);
 
     if (!this.slottedInShell) {
-      document.documentElement.classList.add(CSS.overflowHidden);
+      document.documentElement.style.setProperty("overflow", "hidden");
```

Comment:
In order to play nice with existing inline styles, could you store the value of `overflow` (if set) and restore when our override is removed?